IO/Logging/Logger.cs

```diff
@@ -1,29 +1,53 @@
-﻿namespace LocalAdmin.V2.IO.Logging
+﻿using System;
+using System.IO;
+using System.IO.Compression;
+
+namespace LocalAdmin.V2.IO.Logging
 {
-    /*
     public static class Logger
     {
-        public static void Log(string text)
+        public static TextWriter? Writer { get; private set; }
+
+        public static void Setup(ushort port)
         {
-            if (!File.Exists("localadmin.log")) File.Create("localadmin.log");
+            var directoryPath = Path.Combine("logs", port.ToString());
+            var filePath = Path.Combine(directoryPath, "latest.log");
 
-            using (var streamWriter = new StreamWriter("localadmin.log"))
+            Directory.CreateDirectory(directoryPath);
+            if (File.Exists(filePath))
```

Comment:
Just use File.Move instead, compression shouldn't be done here

Reply:
Why?

Reply:
Just compress logs on creation instead

Reply:
What do you mean?